### PR TITLE
[SDK-3820] Initial beta version of Auth0-Angular

### PR DESCRIPTION
**Pull request by frederikprijck**
Adding the beta quickstart that goes with the soon-to-be-released v2 version of Auth0-Angular.

The only change code-wise is the placement of the `returnTo` parameter when calling logout.

### CODE REVIEW

File: articles/quickstart/spa/angular-beta/01-login.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+})
+export class AuthButtonComponent {
+  constructor(@Inject(DOCUMENT) public document: Document, public auth: AuthService) {}
+}
+```
+
+Specify the `returnTo` option when calling `logout` to tell Auth0 where it should redirect to after a successful logout. This value must be specified in [the **Allowed Logout URLs** setting](#configure-logout-urls) in the dashboard.
+
+:::note
+Here we use `http://localhost:4200` as the value for `returnTo`, but the associate sample uses `window.location.origin`, which in this case would resolve to the same value. Ultimately, this value should point to the root URL for your application.
```
Comment (ewanharris) [inline articles/quickstart/spa/angular-beta/01-login.md]: Should we refer to `returnTo` as `logoutParams.returnTo`?
```suggestion
Here we use `http://localhost:4200` as the value for `logoutParams.returnTo`, but the associate sample uses `window.location.origin`, which in this case would resolve to the same value. Ultimately, this value should point to the root URL for your application.
```

File: articles/quickstart/spa/angular-beta/interactive.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+- you are redirected to Auth0 for authentication.
+- Auth0 successfully redirects back to your application after authentication.
+- you do not receive any errors in the console related to Auth0.
+
+:::
+
+:::checkpoint-failure
+Sorry about that. Here are a few things to double check:
+
+- make sure you configured the correct `redirect_uri`
```
Comment (ewanharris) [inline articles/quickstart/spa/angular-beta/interactive.md]: Should we refer to `redirect_uri` as `authorizationParams.redirect_uri`?
```suggestion
- make sure you configured the correct `authorizationParams.redirect_uri`
```

File: articles/quickstart/spa/angular-beta/01-login.md (hunk 1 above)
Comment (frederikprijck) [reply articles/quickstart/spa/angular-beta/01-login.md]: Yes, thanks!

File: articles/quickstart/spa/angular-beta/interactive.md (hunk 2 above)
Comment (frederikprijck) [reply articles/quickstart/spa/angular-beta/interactive.md]: Is the / a typo, or intentional?

Comment (ewanharris) [reply articles/quickstart/spa/angular-beta/interactive.md]: Typo 🤦🏻

**Review by ewanharris — APPROVED** [review]

**Pull request merged**